Sessions/add session tracks

emb03 opened the pull request:
@z3cka plz merge to stage, Gary wants this up for review ASAP

emb03 committed:
NDLC: adds program section with sessions and tracks filtered on taxonomy

emb03 committed:
NDLC: Adds new section 'conduct', revises section 'program', adds content, adds css

z3cka commented on config/system.yaml:
```diff
@@ -35,9 +35,7 @@ errors:
-  enabled: false
-  twig: false
+  enabled: true
+  twig: true
```
We don't want to turn debugging on in production. @emb03 please remove all the above configs.

emb03 committed:
NDLC: turn off debugging!

z3cka committed:
fixes conflict that PR #78 + #79 created
Merge branch 'NDLC/adds_sessions' of https://github.com/emb03/ndlc.info into emb03-NDLC/adds_sessions

Conflicts:
	pages/00.sessions/From-My-Block-to-Born-on-the-Rez-A-Linguistical-Analysis-of-Lyrics-by-2pac,-Frank-Waln,-Litefoot,-Nataanii-Means,-and-Night-Shield/session.md
	pages/00.sessions/Let-Us-Work-Together-Academic-Libraries-and-Social-Justice-on-Campus/session.md
	pages/00.sessions/Making-the-Invisible-Visible-Diversity,-Dialougue-and-Multicultural-Awareness-Through-Digital-Projects/session.md
	pages/00.sessions/Services-and-Studies-to-Support-First-Generation-College-Students/session.md
	pages/03.program/all-sessions.md
	pages/03.program/program.md
	themes/bootstrapndlc/templates/sessions.html.twig

z3cka committed:
Merge remote-tracking branch 'remotes/gh-eb/NDLC/adds_sessions' into fix_78+79

Commit:
Merge pull request #3 from z3cka/fix_78+79

Fix #78 + #79, merging

z3cka commented:
looks good... has some content cleanup to do.
